ui/core-react/src/core-react/hocs/withOnOutsideClick.tsx

```diff
@@ -75,6 +75,7 @@ export const withOnOutsideClick = <ComponentProps extends {}>(
     /** @internal */
     public handleDocumentPointerDown = (e: PointerEvent) => {
       this.isDownOutside = true;
+      // istanbul ignore else
```

Comment:
This path was tested because of a shallow rendering that didnt unmount and left the handler in the document, I added an ignore because it would be a timing feat to test this otherwise (unmounting the component between the time the handler is called and the time it trigger the event...)